python/katana/dynamic_bitset.pyx

```diff
@@ -0,0 +1,56 @@
+from .cpp.libgalois.DynamicBitset cimport DynamicBitset as CDynamicBitset
+
+
+cdef class DynamicBitset:
+
```

Comment:
Normal Python style doesn't have a space here unless there is a doc string.

Reply:
done